tests/test_worker.py

```diff
@@ -93,3 +96,36 @@ def test_normal_run(self):
         worker.run()
         self.assertEqual(consumer.call_count, 1)
         self.assertEqual(channel_layer.send.call_count, 0)
+
+
+class WorkerGroupTests(ChannelTestCase):
+
+    def test_graceful_stop(self):
+
```

Comment:
Add doc string what this test is actually do.

Reply:
Covered in https://github.com/django/channels/pull/792/commits/713e972bfecb6534c177169c498888ae8322bd0b.